docs/src/docs/execution/execution-fail-annotation.adoc

```diff
@@ -42,13 +42,38 @@ public void testItWillFail() {
 |===
 | Attribute | Description
 | description | Give more details about the failure.
-| ticketId | Define a bug ticket ID as an int value.
-| ticketString | Define a bug ticket ID as a String value.
+| ticketId | Define a bug ticket ID as an int value (`@deprecated`)
+| ticketString | Define a bug ticket ID or URL related to this failure.
 | intoReport | If `true` the failing test is shown as `Failed` instead of `Expected Failed` (default: `false`).
-| validFor | Define the <<Define conditions, conditions>> of the expected failed.
+| validator | Define a method that checks if the expected failure is valid.
+| validatorClass | Define a class for the `validator` method (optional)
+| validFor | Define the <<Define conditions, conditions>> of the expected failed.  (`@deprecated`)
 |===
 
-== Define conditions
+== Defining a validator
```

Comment:
We should also show an example for `validatorClass`  like the example of the integration tests.